theano/configparser.py

```diff
@@ -98,15 +98,17 @@ class change_flags(object):
 
     Useful during tests.
     """
-    def __init__(self, **kwargs):
+    def __init__(self, args=[], **kwargs):
```

Comment:
Don't use a list as a default argument.  I know it doesn't really matter here, but it's a bad habit and a bad example.